lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -627,3 +633,24 @@ func getNetworkSentryConfig(network *configurator.Network) *lte_mconfig.SentryCo
 		UrlPython:    string(sentryConfig.URLPython),
 	}
 }
+
+// getSyncInterval takes network-wide subscriberdb sync interval and overrides it if also set for gateway.
+// If sync interval is unset for both network and gateway, a default is read from lte/cloud/configs/subscriberdb.yml
+func getSyncInterval(nwEpc *lte_models.NetworkEpcConfigs, gwEpc *lte_models.GatewayEpcConfigs, defaultSyncInterval uint32) uint32 {
```

Comment:
Instead of passing in defaultSyncInterval, just make this a method on the obj and use the defaultSyncInterval attribute